packages/@aws-cdk/aws-backup/lib/rule.ts

```diff
@@ -156,5 +163,19 @@ export class BackupPlanRule {
     if (props.scheduleExpression && !/^cron/.test(props.scheduleExpression.expressionString)) {
       throw new Error('`scheduleExpression` must be of type `cron`');
     }
+
+    if (props.enableContinuousBackup && !props.deleteAfter) {
```

Comment:
How would you feel about defaulting `deleteAfter` to the maximum `Duration.days(35)` if not specified? If that is a sensible default to have, I think this would be a better experience for users.

Reply:
Good point. I changed the default to 35  days if `enableContinuousBackup` is enabled.